tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "allowJs": true,
```

Comment:
`allowJs` and `checkJs` are what enable type checking. `noEmit` is added because we don't actually want to compile the files.